src/components/Header/Tests/SearchBar.test.jsx

```diff
@@ -0,0 +1,17 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
You can add a setupTests file with this line so you don't have to add it each tie in you're test files each time. To configure it check [this ref](https://jestjs.io/docs/configuration#setupfiles-array).